### ISSUE TITLE
paddlex 做rec 微调，训练集中的dict.txt该怎么选择

### ISSUE THREAD

**Issue by Double97828** [PaddlePaddle/PaddleX#2579]
## Checklist:

- [ ] 查找[历史相关issue](https://github.com/PaddlePaddle/PaddleX/issues)寻求解答
- [ ] 翻阅[FAQ](https://paddlepaddle.github.io/PaddleX/main/FAQ.html)
- [ ] 翻阅[PaddleX 文档](https://paddlepaddle.github.io/PaddleX/main/index.html)
- [ ] 确认bug是否在新版本里还未修复

## 复现

1. 您使用的命令是？
python main.py -c /home/PaddleX-release-3.0-beta2/paddlex/configs/text_recognition/sfz_ppocrv4_rec.yaml
2. 您使用的数据集是？
PPOCRlabel 自己标标注的训练集
3. 请提供您出现的报错信息及相关log
无报错信息，但是在做check_dataset提示无dict.txt文件，我从paddleocr的项目中找了一个dict用的。但是在做rec的微调训练过程中 ppocr INFO: epoch: [200/200], global_step: 600, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.196965, CTCLoss: 33.463882, NRTRLoss: 1.375257, loss: 34.837538, avg_reader_cost: 0.25354 s, avg_batch_cost: 0.29127 s, avg_samples: 3.45, ips: 11.84484 samples/s, eta: 0:00:00, max_mem_reserved: 3320 MB, max_mem_allocated: 2860 MB
acc一直为0
## 环境

1. 请提供您使用的PaddleX的版本号
/PaddleX-release-3.0-beta2
2. 请提供您使用的操作系统信息，如Linux/Windows/MacOS
linux 
3. 请问您使用的Python版本是
Python 3.10.15

**Comment by changdazhou** [comment]
一般选取paddleOCR中提供的字典文件即可哈，后续我们会为大家提供一个字典生成方法，可以生成自己数据集的字典文件

**Comment by Double97828** [comment]
> 一般选取paddleOCR中提供的字典文件即可哈，后续我们会为大家提供一个字典生成方法，可以生成自己数据集的字典文件

那在做rec的微调中一直出现的lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.196965, CTCLoss: 33.463882, NRTRLoss: 1.375257, loss: 34.837538, avg_reader_cost: 0.25354 s, avg_batch_cost: 0.29127 s, avg_samples: 3.45, ips: 11.84484 samples/s, eta: 0:00:00, max_mem_reserved: 3320 MB, max_mem_allocated: 2860 MB ，为什么 会造成acc一直为0的情况

**Comment by changdazhou** [comment]
你的学习率和批大小如何设置的呢，数据量有多少

**Comment by Double97828** [comment]
> 你的学习率和批大小如何设置的呢，数据量有多少

Train:
  epochs_iters: 200
  batch_size: 32
  learning_rate: 0.0001
  pretrain_weight_path: ./PaddleX-release-3.0-beta2/paddlex/pretrained/rec/PP-OCRv4_server_rec_pretrained.pdparams
  resume_path: null
  log_interval: 20
  eval_interval: 50
  save_interval: 50
  
  
 训练数据是我用PPOCRLabel，训练数据125，验证集18.

**Comment by changdazhou** [comment]
训练数据太少了，你是什么数据呢，是不是用官方模型几乎完全预测不出来那种，如果仅仅是想看看你这批数据的效果，建议pretrain_weight_path设置为None，learning_rate调大一些看看，可以依次尝试0.1,0.05,0.01看看

**Comment by Double97828** [comment]
> 训练数据太少了，你是什么数据呢，是不是用官方模型几乎完全预测不出来那种，如果仅仅是想看看你这批数据的效果，建议pretrain_weight_path设置为None，learning_rate调大一些看看，可以依次尝试0.1,0.05,0.01看看

用的数据是自己标注的身份证，但是用预训练模型其实已经能识别出来很多了，就是想微调修正一下,然后pretrain_weight_path用的就是官方的预训练模型。所以我才在想是不是dict出现问题导致的，因为用paddle的PPOCRLabel标注出来的数据没有字典。但是我修改成0.1就好了，应该是之前lr太小了？感谢

**Comment by changdazhou** [comment]
是的，学习率太小了

**Comment by Double97828** [comment]
> 是的，学习率太小了

还是有问题，我昨天搞错了，我吧dict里面的东西全都删除了，所以acc 是1，等我吧dict 换成paddleocr的那个ppocr/utils/ppocr_keys_v1.txt，acc又一直是0

**Comment by changdazhou** [comment]
试试这里这个：https://paddle-model-ecology.bj.bcebos.com/paddlex/data/ocr_rec_dataset_examples.tar

**Comment by Double97828** [comment]
> 试试这里这个：https://paddle-model-ecology.bj.bcebos.com/paddlex/data/ocr_rec_dataset_examples.tar

还是不行，你们在自己创建数据集训练会遇见这种问题么。。。

**Comment by changdazhou** [comment]
我们不会用这么少的数据去训练，方便提供一下数据集我们验证一下吗

**Comment by Double97828** [comment]
> 我们不会用这么少的数据去训练，方便提供一下数据集我们验证一下吗

搞好了，不是dict问题，就是我的问题。。。我重新标注了一下数据，在训练就好饿了

**Comment by changdazhou** [comment]
好的，那没什么问题我这边先close了哈
